tutorials/casper/Makefile

```diff
@@ -0,0 +1,17 @@
+REDIS_HOME = $(HOME)/sandboxes/SmartRedis
```

Comment:
``SMARTREDIS_HOME`` might disentangle the two references, SmartRedis and Redis(the database)

Reply:
I've refactored this to use the installed SmartRedis module.